setup.py

```diff
@@ -25,7 +25,7 @@
     install_requires=[
         "attrs>=21.3.0",
         "itemadapter>=0.8.0",
-        "web-poet>=0.7.0",
+        "web-poet>=0.14.0",
```

Comment:
I think "min" environment in tox.ini needs to be updated according to this change. Would it make sense to add a comment to setup.py, to ensure we don't forget to do it with other version updates in install_requires?